src/FsAutoComplete/CodeFixes.fs

```diff
@@ -558,22 +581,37 @@ module Fixes =
                     File = codeActionParams.TextDocument
                     Edits =
                       [| { Range = { Start = dash; End = inc lines dash }
-                           NewText = "- " } |] } ]
+                           NewText = "- " } |]
+                    Kind = Fix } ]
           | None -> return []
         }
         |> AsyncResult.foldResult id (fun _ -> []))
       (Set.ofList [ "3" ])
 
   /// a codefix that corrects == equality to = equality
-  let doubleEqualsToSingleEquality: CodeFix =
+  let doubleEqualsToSingleEquality (getFileLines: string -> Result<string [], _>): CodeFix =
```

Comment:
this is actually a logic bugfix: 43 is thrown for any sort of operator error, so we should make sure that the operator that's not supported actually is `==` before suggesting to replace it.